### PR TITLE
Update add-3d-model-babylon.html with comments on occlusion and interactivity

### PR DESCRIPTION
**Pull request by dennemark**
Adding optional code for interactivity and occlusion.

## Launch Checklist

 - [ ] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [ ] Briefly describe the changes in this PR.
 - [ ] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [ ] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

File: test/examples/add-3d-model-babylon.html [hunk 1]
```diff
@@ -77,9 +77,21 @@
                 true
             );
             this.scene = new BABYLON.Scene(this.engine);
+            /**
+            * optionally add 
+            * this.scene.autoClearDepthAndStencil = false
+            * and for renderingGroupIds set this individually via
+            * this.scene.setRenderingAutoClearDepthStencil(1,false)
```
Comment (HarelM) [inline test/examples/add-3d-model-babylon.html:84]: Can we use links to docs instead of comments that will be get out of date?

Comment (dennemark) [reply test/examples/add-3d-model-babylon.html:84]: So far babylon has a pretty consistent API.
We could point to this part of their docs though: https://doc.babylonjs.com/features/featuresDeepDive/scene/optimize_your_scene#reducing-calls-to-glclear

Concerning the preventDefault remarks, I think it is quite an exception for babylonjs so it is not documented except within their api: https://doc.babylonjs.com/typedoc/classes/BABYLON.Scene#preventDefaultOnPointerDown

What is your preference?

Comment (HarelM) [reply test/examples/add-3d-model-babylon.html:84]: I prefer links that might get broken over comments that will get out of date :-)
But either way works for me, just make sure to fix lint issues.

Comment (dennemark) [reply test/examples/add-3d-model-babylon.html:84]: i have added the links but also kept the comments for explanation. since the docs alone won`t help.

**Review by HarelM — APPROVED** [review]

**Comment by HarelM** [comment]
THANKS!

**Pull request merged**